### DISCUSSION TITLE
How to load data in test?

### DISCUSSION THREAD

**Discussion by leon** [as-pect/as-pect#324] [Q&A]
Hello!

I've just started playing around with web assembly and assemblyscript, so bear with me.

I'm trying to write a module that should manipulate an image.
The module is going to get a Uint8ClampedArray passed to it and it should create a new output image.

I was wondering what the best way is to test getting the image data passed a assemblyscript function?

**Comment by jtenner** [comment]
Hello! Did you check out the gitbook link on the main repo documentation? If you need specific help, I'd be glad to offer it.
